README.md

```diff
@@ -142,6 +142,57 @@ because Dependabot can get very noisy) have code owners:
 npm run lint:codeowners
 ```
 
+### Releasing versions
+
+1. Create a new branch called `release/vx.y.z`, where `x.y.z` is the new version
+   number, following [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+1. Update `CHANGELOG.md` to batch the changes in this version under a heading in
```

Comment:
Do you know you've got a lot of `1.`s throughout this section?

Reply:
Yup. That's intentional. Markdown will do the ordering for us, so we don't need to mess with renumbering when adding or removing steps.